test/e2e/proxy.go

```diff
@@ -52,6 +52,7 @@ func proxyContext(version string) {
 	f := NewFramework("proxy")
 	prefix := "/api/" + version
 
+	// Port here has to be kept in sync with default kubelet port.
```

Comment:
This will be no longer true and each Kubelet will be able to have different port.